source/includes/fundamentals/code-snippets/authentication/aws-connection-string-session-token.go

```diff
@@ -0,0 +1,12 @@
+var sessionToken string
```

Comment:
I'm not very familiar with the convention but do we need to list `accessKeyID` as well?